Dockerfile

```diff
@@ -1,6 +1,6 @@
 # build stage
-FROM node:lts-alpine as build-stage
-RUN apk add openjdk11-jre-headless
+FROM node:18-alpine as build-stage
+RUN apk add openjdk17-jre-headless
```

Comment:
To use the same fixed build JDK used within the other repos and locally. Trying to prevent dependency and version conflicts

Reply:
Using solely versions that other repos of More use is at least the following ordered by priority of advantages:

- better DX
- delay of obsolescence in the future
- more possible compatibility for future deps, features, etc.
- hopefully improved performance

in this case. In general, there is additionally:

- new versions incorporate new features
- less errors, more bug fixes
- better security

The PR for a node version update in the PL is here, as I ran into a necessary vite upadate that had no node 16 compatibility https://github.com/MORE-Platform/more-studymanager-frontend/pull/580,

Reply:
Talked in the office, if it works it works and only used for API building here, should be good